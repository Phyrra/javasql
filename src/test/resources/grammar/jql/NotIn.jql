field not in (1, 2, 3)