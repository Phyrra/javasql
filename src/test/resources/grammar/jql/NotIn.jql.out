NOT ([field] IN (
1, 2, 3
))